util/p9/bkreplace

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+timestamp=$(date +"%Y-%m-%d_%H-%M-%S")
+
+echo "backing up gatling-puppet-load-test"
+tar czf ./gplt.${timestamp}.tar.gz ./gatling-puppet-load-test
+
+echo "removing gatling-puppet-load-test"
+rm -rf ./gatling-puppet-load-test
+
+echo "cloning gatling-puppet-load-test"
+git clone https://github.com/puppetlabs/gatling-puppet-load-test.git
```

Comment:
Should this have a target destination path set?

Reply:
Since these scripts were designed to run from an "instance directory" the target is always the current working directory.

Reply:
Added [docs/slv_environment_setup.md](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md) with a [Utility scripts](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md#utility-scripts) section.